Makefile

```diff
@@ -97,8 +97,8 @@ ifndef SHASUMCMD
 endif
 
 .PHONY: kops-install # Install kops to local $GOPATH/bin
-kops-install: gobindata-tool ${BINDATA_TARGETS}
-	go install ${EXTRA_BUILDFLAGS} -ldflags "-X k8s.io/kops.Version=${VERSION} -X k8s.io/kops.GitVersion=${GITSHA} ${EXTRA_LDFLAGS}" k8s.io/kops/cmd/kops/
+kops-install: ${KOPS}
```

Comment:
I'm trying to get any use of "go install" out of the Makefile so that we can keep track of what has, and what has not been built. We lose this insight once we're using "go install" to install things out-of-tree.

Reply:
The problem is that it's slow to have make analyze all the dependencies, and `go build` is slower than `go install` I believe.

I want to keep the default target fast, as that's the one end-users run - and the one I run during development!

Also I was wondering - how should we express the dependency on `${VERSION}`, `${GITSHA}` and `${EXTRA_LDFLAGS}` ? @